ajax/libs/jqPlot/package.json

```diff
@@ -12,10 +12,7 @@
   ],
   "licenses": [
     "MIT",
-    {
-      "type": "GPL",
-      "url": "http://www.opensource.org/licenses/gpl-license.php"
-    }
+    "CC-BY-3.0"
```

Comment:
Let's use `GPL-2.0`
The reference is https://github.com/jqPlot/jqPlot/#legal-notices.
And the commit message need to be modified (cc issue number).
Thank you.

Reply:
There is another license: MIT
You can use (don't forget the comma)
```
"licenses": [
  "GPL-2.0"
  "MIT"
],
```

> jqPlot is currently available for use in all personal or commercial projects under both the **MIT and GPL version 2.0** licenses.

Reply:
@gavin0430 So please help modify this PR, thank you.